README.rst

```diff
@@ -29,13 +29,206 @@
+library management (ILS), institutional repositories, digital libraries and
+research data management. This framework, developed at `CERN`_ is called
+`Invenio`_.
+
+Who’s going to run RERO ILS, in which context?
+----------------------------------------------
+
+RERO ILS is a *free software*, which means that you are free to copy, modify
+and distribute it, as long as you respect the `AGPLv3`_ license.
+
+RERO ILS allows a team of library professionals and IT people to provide
+an ILS *as a Service* to several library networks through a unique
+instance (installation) of the software. This is what we call the
+*consortial model feature*. Each network, or *organisation* as we named it, is
+isolated from the others, but they all share a union catalog, which helps
+patrons discover documents and allows librarian to share cataloguing effort.
+Inside an organisation, libraries share most resources and parameters, without
+preventing specific configurations or restricting the edition of some resources
+to a single library.
+
+RERO ILS is very much adapted to the needs of the RERO *centrale* and its
+partners and customers, both present and future. RERO provides its ILS *as a
+service* to every interested library or organisation. But the software can be
+deployed by other institutions, in different contexts as well, if they can
+afford the required effort on configuration or development. It could be run for
+a standalone library in an instance with only one organisation with a single
+library, but RERO ILS is not designed for that use case. It’s powerful, thus
+complex.
+
+What does RERO ILS?
+---------------------
+
+RERO ILS is still under heavy development, but it already provides a *public
+interface* for visitors and patrons. This interface mainly includes searching
+features within the catalog, along with a set of other patron-oriented
+functionalities. It has multiple views, the *union catalog* view, with all the
+documents of all the organisations, and a view for each organisation with all
+the documents belonging to one of its libraries. Patrons are able to place
+requests on desired items, check their profile, see the list of borrowed items
+and so on.
+
+Librarians use a *professional interface*, in which they perform their daily
+tasks:
+
+-  Administering organisations, libraries, circulation policies.
+-  Managing users: patrons, but also librarians and system librarians.
+-  Performing circulation tasks, such as checkins, checkouts, renewals,
+   managing requests.
+-  Managing patron fees (overdue, subscriptions).
+-  Searching for documents, describing documents, linking authorities to
+   documents, importing metadata in MARC format, adding items to
+   documents.
+-  Setting serials patterns, receiving issues.
+-  Managing acquisition budgets, vendor information, orders and invoices.
+
+The first major release of RERO ILS (``v1.0.0``) is planned for the end of 2020,
```

Comment:
For me this paragraph doesn't belong to *What does RERO ILS*. We could add a section *Rollout and further developments*.

Reply:
@pronguen I've added a *present/future* mention, hoping it would be clearer.

Reply:
It seems perfect, thanks! 👍